core/src/col_metadata.rs

```diff
@@ -8,6 +8,16 @@ pub enum ColumnNullability {
     Unknown,
 }
 
+impl From<ColumnNullability> for Nullability {
```

Comment:
Should we drop our `ColumnNullability` entirely and just use `odbc_sys::Nullability` in our code wherever necessary? Avoid the `From` implementation entirely?

I think at the time I created the `ColumnNullability` type, I mistakenly checked `odbc-sys 0.18` which _does not_ include a `Nullability` type. If I knew it existed in `0.21` I would've just used it from the start.

Reply:
sgtm! I was kind of wondering the same thing, but thought there might be a good reason for it.

Reply:
Fun fact: found out I was actually reporting Nullability incorrectly!